core/update.php

```diff
@@ -323,6 +323,9 @@ function update_info_page() {
   $output .= "<li>Install your new files into the appropriate location, as described in <a href=\"https://backdropcms.org/upgrade\">the handbook</a>.</li>\n";
   $output .= "</ol>\n";
   $output .= "<p>After performing the above steps proceed using the continue button.</p>\n";
+  if ($modules_to_enable = update_upgrade_check_dependencies()) {
```

Comment:
```suggestion
  $modules_to_enable = update_upgrade_check_dependencies();
  if (!empty($modules_to_enable)) {
```
Suggestion to improve code readability. Putting an assignment inside a conditional looks like it might be a typo (`=` vs `==`). Splitting the assignment and test is easier on the human (and no worse for the computer).

Also, since elsewhere `$modules_to_enable` is an array, but here the result of `update_upgrade_check_dependencies()` is a string, I suggest you change the name of the variable here (perhaps something like `$dependencies_msg`?).

Reply:
Thanks @bugfolder. I will include this in the next PR in a minute.